packages/desktop/views/dashboard/wallet/Wallet.svelte

```diff
@@ -2,7 +2,7 @@
     import { selectedWallet } from '@core/wallet'
     import { ImplicitAccountCreationView, WalletView } from './views'
 
-    const hasAccount: boolean = false
+    const hasAccount: boolean = $selectedWallet.accountsOutput.length !== 0
```

Comment:
It will probably benefit from being reactive
```suggestion
    $: hasAccount: boolean = $selectedWallet.accountsOutput.length !== 0
```